packages/data-service-generator/src/server/custom-module/custom-service/service.template.ts

```diff
@@ -0,0 +1,7 @@
+import { Injectable } from "@nestjs/common";
+import { PrismaService } from "../prisma/prisma.service";
+
+@Injectable()
+export class SERVICE {
+  constructor(protected readonly prisma: PrismaService) {}
```

Comment:
Do we need to inject prisma into custom module services?

Reply:
removed